### PR TITLE
remove snappy RPC support

### PR DESCRIPTION
**Pull request by arnetheduck**
removed in 0.12.2 - the flow, in particular when the other peer doesn't
support snappy, is hard to follow because of the trial-and-error
approach - removing it simplifies things and removes some of the
hard-to-read parts of the thunking etc

### CODE REVIEW

**Comment by protolambda** [comment]
Uhh, you mean "remove non-snappy RPC support?

**Comment by zah** [comment]
Yes, I got initially confused as well :)

**Comment by arnetheduck** [comment]
> Uhh, you mean "remove non-snappy RPC support?

at least somebody is reading the pr :)
